cluster.go

```diff
@@ -277,35 +277,35 @@ func NewCluster() *Cluster {
 		EventReceiver: NopEventReceiver,
 
 		joiningLeavingNodes: make(chan nodeAction, 10), // buffered channel
-		jobs:                make(map[int64]*ResizeJob),
+		jobs:                make(map[int64]*resizeJob),
 		closing:             make(chan struct{}),
 		joining:             make(chan struct{}),
 
 		Logger: NopLogger,
 	}
 }
 
-// Coordinator returns the coordinator node.
-func (c *Cluster) CoordinatorNode() *Node {
-	return c.nodeByID(c.Coordinator)
+// coordinatorNode returns the coordinator node.
+func (c *Cluster) coordinatorNode() *Node {
+	return c.unprotectedNodeByID(c.Coordinator)
```

Comment:
OK, I think we're missing a lot of locking in Cluster.... I know this PR is a pretty straight up renaming, so I'll ignore it for now.

Reply:
agreed. i’ll make a ticket for that